pwndbg/gdblib/nearpc.py

```diff
@@ -162,7 +165,19 @@ def nearpc(pc=None, lines=None, emulate=False, repeat=False) -> List[str]:
             symbol = C.highlight(symbol)
             first_pc = False
 
-        line = " ".join((prefix, address_str, symbol, asm))
+        if show_opcodes_size > 0:
+            opcodes = instr.bytes.hex()
+            opcodes = opcodes[: show_opcodes_size * 2]
+            align = show_opcodes_size * 2 + 10
+            if len(instr.bytes) > show_opcodes_size:
+                opcodes += pwndbg.color.gray("...")
+                align += 9  # len(pwndbg.color.gray(""))
```

Comment:
Can you add some comments explaining how the alignment is calculated? So if we show 9 opcode bytes (18 characters) we'll pad the end of it by 10 spaces, but if we add a '...' we pad the end by 19 spaces?

Reply:
> Can you add some comments explaining how the alignment is calculated?

Sure.

> So if we show 9 opcode bytes (18 characters) we'll pad the end of it by 10 spaces, but if we add a '...' we pad the end by 19 spaces?

This 9 byte is because the gray color uses 9 invisible characters, so we need to extend the alignment for it to make it look the same.